code/game/machinery/computer/surgery_television.dm

```diff
@@ -0,0 +1,161 @@
+#define SURGERY_EXP_LEVEL_1 3
+#define SURGERY_EXP_LEVEL_2 7
+#define SURGERY_EXP_LEVEL_3 12
+
+#define SURGERY_UNTRAINED 0
+#define SURGERY_FAMILIAR 1
+#define SURGERY_TRAINED 2
+#define SURGERY_EXPERT 3
+
+/obj/machinery/computer/surgery_television
```

Comment:
i feel like a generic television that can be used for potential other skills in the future would be a big step up from a "surgery television" specific to only surgery. making it specific like this is a really weird decision imo

Reply:
I just went with a TV as a medium because I thought of that one thread on the forums about watching surgery videos on youtube, and figured dissection recordings would be a flavorful way of introducing surgery studying that also fits with the research network. I'm not sure television screens would be suitable for teaching other skills, but at the very least adding a base television that can give different benefits like moodies would be a good direction